apps/editing-toolkit/editing-toolkit-plugin/coming-soon/coming-soon.php

```diff
@@ -159,47 +125,7 @@ function coming_soon_page() {
 	add_filter( 'wpcom_disable_logged_out_follow', '__return_true', 10, 1 ); // Disable follow actionbar.
 	add_filter( 'wpl_is_enabled_sitewide', '__return_false', 10, 1 ); // Disable likes.
 
-	$should_show_fallback = false;
```

Comment:
Could we also remove the cookie banner?

```
add_filter( 'jetpack_disable_eu_cookie_law_widget', '__return_true', 1 );
```

I don't think we're setting any cookies on the default coming soon page

Reply:
I'll double check that before disabling. Should be able to see in the debug tools what cookies are being set by the site. I think you're right that we won't be _using_ any of the cookies this banner's talking about (because we're not showing ads), but that doesn't mean we're not still sending them.

Reply:
Fair point.

I did check in the debug tools and in the cookie extension. No cookies were set for the domain, e.g., testsite.wordpress.\com, and I don't think they're sent x-domain either? That is for wordpress.com.

Reply:
Actually, while we don't set any cookies on the v2 page (yet), we might.

The v1 coming soon page does set a test cookie to check if the browser supports them (so we can auto-sign them in).

If we do that then we should keep the cookie banner 👍